doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -77,38 +77,41 @@ Run GeoServer from IntelliJ
    .. image:: img/intellij_run_config.png
       :width: 600
 
+#. It's possible now to run GeoServer by selecting ``Run -> Run 'Start'``
+
+   .. image:: img/intellij_run_start.png
+      :width: 300
+
+Troubleshooting
+^^^^^^^^^^^^^^^
+
 #. If there are errors such as "cannot find symbol class ASTAxisId", some generated code is not being included in the build.  Using ``wcs1_1`` as the working directory, run a ``mvn clean install``.
-#. If you get a compiler error like ``java.lang.NoSuchMethodError``, it might be due to ``Error Prone``. This tool is switched off by default, but sometimes it turns on after import to IntelliJ. There are two options to fix it:
+#. In the case of compiler errors like ``java.lang.NoSuchMethodError``, it might be due to ``Error Prone``. This tool is switched off by default, but sometimes it turns on after import to IntelliJ. There are two options to fix it:
     #. Go to Maven tool window and uncheck the ``errorprone`` profile, then click ``Reimport All Maven Projects``:
 
        .. image:: img/intellij_maven_errorprone.png
           :width: 400
 
-    #. If you want to use ``errorprone``, notably to perform the QA checks, install the ``Error Prone Compiler`` plugin, restart the IDE and set ``Javac with error-prone`` as a default compiler for the project. Please note that this will slower the build.
+    #. To use ``errorprone``, notably to perform the QA checks, install the ``Error Prone Compiler`` plugin, restart the IDE and set ``Javac with error-prone`` as a default compiler for the project. Please note that this will slower the build.
```

Comment:
```suggestion
    #. To use ``errorprone``, notably to perform the QA checks, install the ``Error Prone Compiler`` plugin, restart the IDE and set ``Javac with error-prone`` as a default compiler for the project. Please note that this will slow down the build.
```